docs/changelog.md

```diff
@@ -2,6 +2,26 @@
 
 ## CARET
 
+### v0.5.11 <small>\_ Sep 30, 2024</small> {id = "0.5.11"}
+
+- **Update**: Fixed a problem where only one legend is displayed even if the maximum number of legends is exceeded. ([caret_analyze #523](https://github.com/tier4/caret_analyze/pull/523))
+
+- **Update**: Fixed mypy1.11 error untyped-overrides. ([caret_analyze #527](https://github.com/tier4/caret_analyze/pull/527))
+
+- **Update**: Fixed stackedbar when time reversal occurs. ([caret_analyze #517](https://github.com/tier4/caret_analyze/pull/517))
```

Comment:
Please use stacked bar graph or `StackedBar`.

Reply:
Fixed in https://github.com/tier4/caret_doc/pull/356/commits/ad0751c8e11b35fc512449ec0dfb7c6656a329eb.